src/main/java/kitchenpos/remote/purgomalum/PurgomalumClient.java

```diff
@@ -1,4 +1,4 @@
-package kitchenpos.infra;
+package kitchenpos.remote.purgomalum;
```

Comment:
모든 컨텍스트들은 아래와 같이 패키지가 나뉘어져 있습니다. 여러 컨텍스트에서 사용되는 비속어 관련 클래스라곤 하지만 기존 패키지 구조를 따라가는게 어떨까요?

- api
- application
- domain
- infra

```suggestion
package kitchenpos.remote.infra.purgomalum;
```